elixir/apps/domain/test/domain/auth_test.exs

```diff
@@ -848,94 +718,43 @@ defmodule Domain.AuthTest do
       assert token1.secret_salt != token2.secret_salt
     end
 
-    test "generates unique secret_hash for same fragment with different salt" do
+    test "generates unique secret_hash for same nonce with different salt" do
```

Comment:
These tests have changed slightly - we no longer expose the `secret_fragment` for callers to set, only `secret_nonce`.